packages/rainbowkit/src/components/ConnectButton/abbreviateNumber.test.ts

```diff
@@ -0,0 +1,55 @@
+import { describe, expect, it } from 'vitest';
+import { abbreviateNumber } from './abbreviateNumber';
+
+const K = 1000;
+const M = 1000 * K;
+const B = 1000 * M;
+
+describe('abbreviateNumber', () => {
+  it('passes through small numbers', () => {
+    expect(abbreviateNumber(1)).toEqual('1');
+    expect(abbreviateNumber(12)).toEqual('12');
+    expect(abbreviateNumber(123)).toEqual('123');
+    expect(abbreviateNumber(1234)).toEqual('1,234');
+    expect(abbreviateNumber(9999)).toEqual('9,999');
+    expect(abbreviateNumber(9999.1)).toEqual('9,999.1');
```

Comment:
Gets a little long here — we cool with this?

Reply:
Yea i think that's fine